spec/integration/support/serve_examples.rb

```diff
@@ -27,7 +27,7 @@
     def test_basic_html(port)
       wait_time = 0
       loop do
-        curl_command = @machinery.run_command("curl http://localhost:#{port}/opensuse131")
+        curl_command = @machinery.run_command("curl http://127.0.0.1:#{port}/opensuse131")
```

Comment:
No, it doesn't really matter I guess. It just wasn't clear to me if/why this change was necessary.